target/linux/imx/patches-6.6/312-dts-imx7d-pico-pi-add-OV5645-camera-support.patch

```diff
@@ -0,0 +1,85 @@
+From e27f93355164be4a637280966cc2de08c32e39ba Mon Sep 17 00:00:00 2001
+From: Lech Perczak <lech.perczak@gmail.com>
+Date: Wed, 11 May 2022 21:02:05 +0200
+Subject: [PATCH] dts: imx7d-pico-pi: add OV5645 camera support
+
```

Comment:
We don't want to maintain this indefinitely, so send it upstream and provide a link here.

Reply:
I wished for some wider testing here, or help figuring out actual stream first, but this board doesn't seem all that popular - so  I'll send it to LKML.